tests/queries/0_stateless/02116_tuple_element.sql

```diff
@@ -17,8 +17,10 @@ EXPLAIN SYNTAX SELECT tupleElement(t1, 'a') FROM t_tuple_element;
 SELECT tupleElement(number, 1) FROM numbers(1); -- { serverError ILLEGAL_TYPE_OF_ARGUMENT }
 SELECT tupleElement(t1) FROM t_tuple_element; -- { serverError NUMBER_OF_ARGUMENTS_DOESNT_MATCH }
 SELECT tupleElement(t1, 'b') FROM t_tuple_element; -- { serverError NOT_FOUND_COLUMN_IN_BLOCK, UNKNOWN_IDENTIFIER }
-SELECT tupleElement(t1, 0) FROM t_tuple_element; -- { serverError ILLEGAL_INDEX }
-SELECT tupleElement(t1, 3) FROM t_tuple_element; -- { serverError ILLEGAL_INDEX }
+SELECT tupleElement(t1, 0) FROM t_tuple_element SETTINGS allow_experimental_analyzer = 0; -- { serverError ILLEGAL_INDEX }
```

Comment:
Note that you can pass several errors in the comment to be accepted, e.g. `-- { serverError ILLEGAL_INDEX, NOT_FOUND_COLUMN_IN_BLOCK }`. That way you don't need to manually test with and without the analyzer (and leave that to global config)

Reply:
Good idea. I actually saw this syntax yesterday in some other test and wondered if the author made a mistake :-)